src/main/java/Duke.java

```diff
@@ -1,10 +1,123 @@
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        initDuke();
+        String lineLogo = "-----------------------------";
+        Scanner in = new Scanner(System.in);
+        String line = " ";
+        String command = " ";
+        String commandSubject = " ";
+        String adverb = " ";
+        String timeline = " ";
+        ArrayList<Task> taskArrayList = new ArrayList<>();
```

Comment:
I think you could initialize as "ArrayList\<Task\> taskArraylist = new ArrayList\<Task\>();" to be more clear.

Reply:
Replaced the ArrayList data structure with a Task[100] array instead